include/seqan3/search/search.hpp

```diff
@@ -138,43 +138,35 @@ struct search_configuration_validator
     /*!\brief Validates the error configuration.
      *
      * \tparam configuration_t The type of the search configuration.
+     * \tparam queries_t       The type of the search queries; must model std::ranges::forward_range.
      *
-     * \param[in] cfg The configuration to validate.
+     * \param[in] cfg     The configuration to validate.
```

Comment:
```suggestion
     * \param[in] cfg The configuration to validate.
```

i've seen this now a lot in your PR. Please try to avoid these indentations since they only look good the first time. As soon as someone renames something with search and replace this will be broken. We run into it very often and it always holds back PRs because one has to fix this afterwards.